Non-breaking error in GitHub Actions validate examples job

A snip of the warning. It is likely due to an outdated version of `pygments` package being installed.

More information can be found on https://github.com/vikinganalytics/mvg/runs/4988855088?check_suite_focus=true

```
Running notebooks containing examples
Input Notebook:  ./docs/source/content/examples/0-check_version.ipynb
Output Notebook: ./docs/source/content/examples/0-check_version.ipynb

Executing:   0%|          | 0/6 [00:00<?, ?cell/s]Executing notebook with kernel: python3

Executing:  17%|█▋        | 1/6 [00:02<00:10,  2.01s/cell]
Executing:  33%|███▎      | 2/6 [00:02<00:03,  1.12cell/s]
Executing:  67%|██████▋   | 4/6 [00:02<00:00,  2.05cell/s][IPKernelApp] ERROR | Exception in message handler:
Traceback (most recent call last):
  File "/home/runner/.local/lib/python3.8/site-packages/IPython/core/interactiveshell.py", line 3251, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "/tmp/ipykernel_1653/3978261553.py", line 1, in <module>
    session.check_version()
  File "/home/runner/work/mvg/mvg/mvg/mvg.py", line 209, in check_version
    raise UserWarning(
UserWarning: API version 0.3.0 may contain features not supporeted by current MVG version 0.11.3

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/.local/lib/python3.8/site-packages/IPython/core/interactiveshell.py", line 1934, in showtraceback
    stb = value._render_traceback_()
AttributeError: 'AssertionError' object has no attribute '_render_traceback_'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/.local/lib/python3.8/site-packages/ipykernel/kernelbase.py", line 357, in dispatch_shell
    await result
  File "/home/runner/.local/lib/python3.8/site-packages/ipykernel/kernelbase.py", line 652, in execute_request
    reply_content = await reply_content
  File "/home/runner/.local/lib/python3.8/site-packages/ipykernel/ipkernel.py", line 353, in do_execute
    res = shell.run_cell(code, store_history=store_history, silent=silent)
  File "/home/runner/.local/lib/python3.8/site-packages/ipykernel/zmqshell.py", line 532, in run_cell
    return super().run_cell(*args, **kwargs)
  File "/home/runner/.local/lib/python3.8/site-packages/IPython/core/interactiveshell.py", line 2768, in run_cell
    result = self._run_cell(
  File "/home/runner/.local/lib/python3.8/site-packages/IPython/core/interactiveshell.py", line 2819, in _run_cell
    self.showtraceback(running_compiled_code=True)
  File "/home/runner/.local/lib/python3.8/site-packages/IPython/core/interactiveshell.py", line 1936, in showtraceback
    stb = self.InteractiveTB.structured_traceback(etype,
  File "/home/runner/.local/lib/python3.8/site-packages/IPython/core/ultratb.py", line 1105, in structured_traceback
    return FormattedTB.structured_traceback(
  File "/home/runner/.local/lib/python3.8/site-packages/IPython/core/ultratb.py", line 999, in structured_traceback
    return VerboseTB.structured_traceback(
  File "/home/runner/.local/lib/python3.8/site-packages/IPython/core/ultratb.py", line 852, in structured_traceback
    formatted_exception = self.format_exception_as_a_whole(etype, evalue, etb, number_of_lines_of_context,
  File "/home/runner/.local/lib/python3.8/site-packages/IPython/core/ultratb.py", line 786, in format_exception_as_a_whole
    records = self.get_records(etb, number_of_lines_of_context, tb_offset)
  File "/home/runner/.local/lib/python3.8/site-packages/IPython/core/ultratb.py", line 830, in get_records
    style = stack_data.style_with_executing_node(style, "bg:ansiyellow")
  File "/home/runner/.local/lib/python3.8/site-packages/stack_data/core.py", line 412, in style_with_executing_node
    class NewStyle(style):
  File "/usr/lib/python3/dist-packages/pygments/style.py", line 91, in __new__
    ndef[4] = colorformat(styledef[3:])
  File "/usr/lib/python3/dist-packages/pygments/style.py", line 58, in colorformat
    assert False, "wrong color format %r" % text
AssertionError: wrong color format 'ansiyellow'
```

--- a/.github/workflows/ci-checks.yml
+++ b/.github/workflows/ci-checks.yml
@@ -84,6 +84,10 @@ jobs:
         name: "Validate examples in documentation"
         steps:
         -   uses: actions/checkout@v2
+        -   name: "Set up Python 3"
+            uses: actions/setup-python@v2
+            with: 
+                python-version: '3.x'
         -   name: "Run all examples"
             env:
                 TEST_TOKEN: ${{ secrets.TEST_TOKEN }}

--- a/mvg/mvg.py
+++ b/mvg/mvg.py
@@ -55,7 +55,7 @@ class MVGAPI:
         self.endpoint = endpoint
         self.token = token
 
-        self.mvg_version = self.parse_version("v0.11.4")
+        self.mvg_version = self.parse_version("v0.11.5")
         self.tested_api_version = self.parse_version("v0.3.0")
 
         # Get API version

